src/__tests__/dependency-resolution.js

```diff
@@ -0,0 +1,286 @@
+/* @flow */
+import tape from 'tape-cup';
+import ClientAppFactory from '../client-app';
+import ServerAppFactory from '../server-app';
+import {withDependencies} from '../with-dependencies';
+import {withMiddleware} from '../with-middleware';
+
+function createToken(name) {
```

Comment:
Shouldn't `createToken` be exposed as part of the core API?

Reply:
I was planning on having this separated into a `fusion-tokens` library and have it colocated with the default tokens